Makefile

```diff
@@ -88,86 +74,98 @@ protobuf_out := $(BUILD_DIR)/plugin-node/proto/msg.pb.cc
 protobuf: $(protobuf_out)
 
 $(protobuf_out): $(PROTO)/proto/msg.proto
-	mkdir -p .build/plugin-node
-	protoc --cpp_out=.build/plugin-node -I $(PROTO) $<
+	mkdir -p $(BUILD_DIR)/plugin-node
+	protoc --cpp_out=$(BUILD_DIR)/plugin-node -I $(PROTO) $<
 
 # Tests
 # -----
 
+TEST          = kiele
+TEST_ASSEMBLE = ./assemble-iele-test
+TEST_BACKEND  = standalone
+TEST_MODE     = NORMAL
+TEST_SCHEDULE = DEFAULT
+TEST_PORT     = 10000
```

Comment:
here is another case where we have taken a step backwards and picked a statically numbered port.